src/commands/org/list/metadata.ts

```diff
@@ -62,7 +62,11 @@ export class ListMetadata extends SfCommand<ListMetadataCommandResult> {
       fs.writeFileSync(flags['output-file'], JSON.stringify(listResult, null, 2));
       this.logSuccess(`Wrote result file to ${flags['output-file']}.`);
     } else if (listResult?.length) {
-      this.styledJSON(listResult);
+      this.table(listResult, Object.fromEntries(Object.keys(listResult[0]).map((k) => [k, { header: k }])), {
```

Comment:
FileProperties also has 2 potentially undefined fields.  If the first item doesn't have them, those columns will be missing from all the other items that might